src/types/search.types.ts

```diff
@@ -20,11 +20,20 @@ export interface QueryObject {
   offset: number;
   limit: number;
   institutions: string[];
-  resourceType: string[];
+  resourceTypes: string[];
   licenses: string[];
   keywords: string[];
   queryFromURL: boolean;
   allowSearch: boolean; //instead of setting queryobject null at initiation
+  showInaccessible: boolean;
+  orderBy: string;
```

Comment:
Nå som queryObject har fått enda flere parameter må vi kanskje tenke på om det blir behov for å nullstille dem ved oppdatering av andre parametre

Reply:
offset, og limit bør vel antageligvis nullstilles hver gang query endres, men resten er jeg litt usikker på om bør nullstilles.

Reply:
Limit ser ut som er en konstant, så da har jeg fått lagt inn at offset nullstilles hver gang query endrer seg

Reply:
nvm, ser jeg har en bug

Reply:
Lagd egen task på dette